python/cugraph/cugraph/dask/structure/mg_property_graph.py

```diff
@@ -667,13 +734,14 @@ def edge_props_to_graph(self,
         # FIXME: MNMG Graphs required renumber to be True due to requirements
         # on legacy code that needed segment offsets, partition offsets,
         # etc. which were previously computed during the "legacy" C
-        # renumbering.  The workaround is to pass renumber=True, then manually
-        # call G.compute_renumber_edge_list(legacy_renum_only=True) to compute
-        # the required meta-data without changing vertex IDs.
+        # renumbering.  The workaround is to always pass renumber=True, but set
+        # legacy_renum_only based on if actual renumbering (ie. changing the
+        # vertex IDs) should happen or not.
+        renumber = True
         if renumber_graph is False:
-            renumber = True
+            legacy_renum_only = True
         else:
-            renumber = renumber_graph
+            legacy_renum_only = False
```

Comment:
I think most of the plc algorithms fail with a memory error if legacy_renum_only=False.  Should we just always make it true for now since the property graph is really only being used for GNNs at this point?

Reply:
Thanks to a long offline conversation with @alexbarghi-nv and @jnke2016 , I think this is resolved. Please review though to make sure I got it right.